cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -236,7 +234,9 @@ public void drop(String name) throws Exception {
 
   @Override
   public void upgrade(String tableName, Properties properties) throws Exception {
```

Comment:
Ok. I'm not sure why it was public on the interface.
Maybe the designer of interface wanted to allow also upgrading of specific tables, instead of all (if all is too many).